.github/native-tests.json

```diff
@@ -45,7 +45,7 @@
         {
             "category": "Data7",
             "timeout": 65,
```

Comment:
Should we increase it a bit?

Reply:
It does not seem necessary: `Quarkus CI / Native Tests - Data7 (pull_request) Successful in 25m`